client/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { StyledFooter } from './components.styled';
+import { FooterCenter } from './FooterCenter';
+
+import { FooterLeft } from './FooterLeft';
+import { FooterRight } from './FooterRight';
+
+export const Footer = () => (
+  <StyledFooter>
+    <FooterLeft />
```

Comment:
again is it nessecary to break it up so much?

Reply:
oky